lr_face/experiments.py

```diff
@@ -193,3 +194,30 @@ def data_keys(self) -> List[str]:
         :return: List[str]
         """
         return list(set(k for v in DATA['all'].values() for k in v.keys()))
+
+
+def dataset_pairs(dataset):
+    """
+    Either get the predefined pairs or make new pairs of the dataset
+    """
+    pairs = dataset.pairs
+    if not pairs:
+        pairs = make_pairs(dataset)
```

Comment:
more elegant would be to put this in the parent class and let classes overwrite if they wish (this was the behaviour some commits ago)